src/components/IndividualReviewCard.jsx

```diff
@@ -38,15 +38,17 @@ const IndividualReview = () => {
           <p>Designer: {singleReview.designer}</p>
           <p>Owner: {singleReview.owner}</p>
           <p>Review: {singleReview.review_body}</p>
-          <div>
+          <Link to={`/reviews/${singleReview.review_id}/comments`}>
+            <button className="comments">View All Comments</button>
```

Comment:
A `Link` is already clickable so you can remove the button tag here as it's redundant. If you want the link to be styled differently add a className to it 😁